### PR TITLE
after respawn select last weapon, muzzle and mode

### PR DESCRIPTION
**Pull request by bux578**
#2138

### CODE REVIEW

File: addons/common/functions/fnc_getAllGear.sqf [hunk 1]
```diff
@@ -47,5 +49,6 @@ if (isNull _unit) exitWith {[
     secondaryWeapon _unit, secondaryWeaponItems _unit, secondaryWeaponMagazine _unit,
     handgunWeapon _unit, handgunItems _unit, handgunMagazine _unit,
     assignedItems _unit,
-    binocular _unit
+    binocular _unit,
+    [currentWeapon _unit, currentMuzzle _unit, currentWeaponMode _unit]
```
Comment (jokoho48) [inline addons/common/functions/fnc_getAllGear.sqf]: currentThrowable?

Comment (bux578) [inline addons/common/functions/fnc_getAllGear.sqf]: Good idea.

Comment (bux578) [inline addons/common/functions/fnc_getAllGear.sqf]: After investigating this is very complicated and probably not worth.

**Comment by bux578** [comment]
Ready for merge.

**Pull request merged**